samples/package.json

```diff
@@ -3,10 +3,14 @@
   "private": true,
   "scripts": {
     "preinstall": "npx only-allow pnpm",
-    "build:js-samples": "concurrently npm:build:js-chatbot npm:build:js-coffee-shop npm:build:js-character-generator",
+    "build:js-angular": "cd js-angular && npm run setup && npm run build",
+    "build:js-character-generator": "cd js-character-generator && npm install && npm run build",
     "build:js-chatbot": "cd js-chatbot && npm install && npm run setup && npm run build",
     "build:js-coffee-shop": "cd js-coffee-shop && npm install && npm run build",
-    "build:js-character-generator": "cd js-character-generator && npm install && npm run build"
+    "build:js-menu": "cd js-menu && npm install && npm run build",
+    "build:js-prompts": "cd js-prompts && npm install && npm run build",
+    "build:js-schoolAgent": "cd js-schoolAgent && npm install && npm run build",
+    "build:js-samples": "concurrently npm:build:js-angular npm:build:js-character-generator npm:build:js-chatbot npm:build:js-coffee-shop npm:build:js-menu npm:build:js-prompts npm:build:js-schoolAgent"
```

Comment:
Is your intention to build all available samples? I think you can generic-ize this with a regex if you wanted.

Reply:
yes, let me look at that

Reply:
pnpm makes it easy to do (e.g. https://github.com/firebase/genkit/blob/main/js/package.json#L10), but not sure about npm...

Reply:
regex'd